browser/ui/webui/basic_ui.cc

```diff
@@ -19,9 +19,20 @@ content::WebUIDataSource* CreateBasicUIHTMLSource(
     const std::string& name,
     const GritResourceMap* resource_map,
     size_t resource_map_size,
-    int html_resource_id) {
+    int html_resource_id,
+    bool disable_trusted_types_csp) {
   content::WebUIDataSource* source =
       content::WebUIDataSource::Create(name);
+  // Some parts of Brave's UI pages are not yet migrated to work without doing
+  // assignments of strings directly into |innerHTML| elements (i.e. see usage
+  // of |dangerouslySetInnerHTML| in .tsx files). This will break Brave due to
+  // committing a Trusted Types related violation now that Trusted Types are
+  // enforced on WebUI pages (see crrev.com/c/2234238 and crrev.com/c/2353547).
+  // We should migrate those pages not to require using |innerHTML|, but for now
```

Comment:
is there a ticket open for this? cc @brave/sec-team

Reply:
There is https://github.com/brave/brave-browser/issues/11845, which was closed because of the PRs that landed in https://github.com/brave/brave-core/pull/6646 and https://github.com/brave/brave-core/pull/6669, which dealt with what we thought was all the instances in Brave.

However, there's still an issue on `shared_vars_css.m.js` files, which are generated using polymer, do still get generated in a way that would assign text to `innerHTML`, which is why we can't remove this particular patch yet.

So, I think we should probably reopen https://github.com/brave/brave-browser/issues/11845, which was kind of the master bug, and comment this in there, but I'd rather pull @diracdeltas to clarify in case a new issue is preferred.

Reply:
i'll reopen that issue, thanks